src/main/java/org/terasology/dynamicCities/region/RegionEntityManager.java

```diff
@@ -123,8 +120,8 @@ public EntityRef get(Vector2ic position) {
 
     public EntityRef getNearest(Vector2ic position) {
         Map<Vector2i, EntityRef> regionEntities = regionEntitiesComponent.regionEntities;
-        int x = Chunks.toChunkPosX(position.x()) * Chunks.SIZE_X + ((Chunks.SIZE_X / 2) - 1);
-        int y = Chunks.toChunkPosZ(position.y()) * Chunks.SIZE_Z + ((Chunks.SIZE_Z / 2) - 1);
+        int x = Chunks.toChunkPosX(position.x()) * Chunks.SIZE_X + Chunks.SIZE_X / 2;
+        int y = Chunks.toChunkPosZ(position.y()) * Chunks.SIZE_Z + Chunks.SIZE_Z / 2;
```

Comment:
:warning: This is a real change, not just punctuation for CheckStyle.

This makes the results line up with what the tests have always expected, contrary to the results produced by #81.

Reply:
Have you checked in MetalRenegades that cities and buildings still show up?

Reply:
Have you checked in MetalRenegades that cities and buildings still show up?

Reply:
I've reverted this change, as it makes the placeParcel test fail.